exercises/concept/inventory-management/.docs/hints.md

```diff
@@ -4,32 +4,32 @@
 
 - [The Python Dictionary Tutorial](https://docs.python.org/3/tutorial/datastructures.html#dictionaries) can be a great introduction.
 
-## 1. Create the dictionary
+## 1. Create an inventory based on a list
```

Comment:
This concept exercise is about lists or dictionaries?  There is a difference in Python, and so we should be clear.